packages/manager/modules/pci/src/projects/project/kubernetes/add/add.controller.js

```diff
@@ -31,6 +33,10 @@ export default class {
       id: null,
       name: this.$translate.instant('kubernetes_add_private_network_none'),
     };
+    this.constants = {
+      nameMaxLength: NAME_INPUT_CONSTRAINTS.MAX_LENGTH,
+      pattern: NAME_INPUT_CONSTRAINTS.PATTERN,
+    };
```

Comment:
May be you can remove above constants declaration and move below code to constructor. 
 this.constants = NAME_INPUT_CONSTRAINTS;

Reply:
Done